cmd/bucket-handlers.go

```diff
@@ -494,6 +494,13 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 		writeErrorResponse(ctx, w, errorCodes.ToAPIErr(s3Error), r.URL, guessIsBrowserReq(r))
 		return
 	}
+	if objectLockEnabled {
```

Comment:
Is this correct? What if the user only has permission to create the bucket, but another user can turn on the default lock?

Reply:
@kannappanr, yes that is possible. Removed this check.